installer/install.sh

```diff
@@ -1,9 +1,12 @@
-#!/bin/bash -ex
+#!/bin/bash -e
 USAGE="Usage: ./install.sh ENVIRONMENT VAULT_TOKEN"
 ENVIRONMENT=${1:?$USAGE}
-VAULT_TOKEN=${2:?$USAGE}
-GIT_URL=`git config --get remote.origin.url`
+export VAULT_TOKEN=${2:?$USAGE}
+export VAULT_ADDR=https://vault.lsst.codes
+VAULT_PATH_PREFIX=`yq -r .vault_path_prefix ../science-platform/values-$ENVIRONMENT.yaml`
+ARGOCD_PASSWORD=`vault kv get --field=argocd.admin.plaintext_password $VAULT_PATH_PREFIX/enclave`
```

Comment:
Not sure I like literal "enclave"; in lsstvaultutils I used that as the term for "an autonomously managed space in vault".   Not sure what would be better.  "_secret_stash" ?

Reply:
any better idea for a name?  Maybe Russ?  I honestly don't care but secret stash sounds like a different secret stash I have.

Reply:
What if we just call it `installer`?